File: go/1748_Sum_of_Unique_Elements/1748_Sum_of_Unique_Elements.go
func sumOfUnique(nums []int) int {
    m := make(map[int]int)
    for _, n := range nums {
        m[n] += 1
    }
    total := 0
    for n := range m {
        if m[n] == 1 {
            total += n
        }
    }
    return total
}

File: java/1748_Sum_of_Unique_Elements/1748_Sum_of_Unique_Elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int[] values = new int[101];
        for (int i : nums) {
            values[i] += 1;
        }
        for (int i = 1; i < values.length; i++) {
            if (values[i] == 1) {
                values[0] += i;
            }
        }
        return values[0];
    }
}
